src/daft-logical-plan/src/ops/agg.rs

```diff
@@ -37,7 +37,10 @@ impl Aggregate {
         let upstream_schema = input.schema();
 
         let groupby_resolver = ExprResolver::default();
-        let agg_resolver = ExprResolver::builder().in_agg_context(true).build();
+        let agg_resolver = ExprResolver::builder()
+            .in_agg_context(true)
+            .groupby(HashSet::from_iter(groupby.clone()))
+            .build();
```

Comment:
I thought about that. We would essentially need groupby to be an `Option<Vec<ExprRef>>` since there can be aggs that don't have groupbys. I think I could do that.

Reply:
couldn't we keep the `in_agg_context` for the non grouped aggs, but just call it inside the `groupby` builder.

so something like 

```rs
impl ExprResolver {
  fn groupby(self, grouping: _) -> Self {
    let mut this = self.in_agg_context(true);
    this.groupby = grouping;
    this
  }
}
```

I think that would work with both cases, and avoid an extra `.in_agg_context` call when building the resolver.